design/features/live-draft/version-1.md

```diff
@@ -0,0 +1,187 @@
+# Live and draft feature v1
+## Status
+
+Date created: *2022-05-26*  
+
+## Contents
+
+- [Live and draft feature v1](#live-and-draft-feature-v1)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [User steps](#user-steps)
+  - [Decisions](#decisions)
+  - [Designs](#designs)
+    - [GOV.UK Forms landing page](#govuk-forms-landing-page)
+    - [Live form view only](#live-form-view-only)
+    - [Your questions view only](#your-questions-view-only)
+    - [Edit draft form task list](#edit-draft-form-task-list)
+      - [Edit draft in progress task list](#edit-draft-in-progress-task-list)
+      - [Edit draft completed task list](#edit-draft-completed-task-list)
+    - [Make your changes live](#make-your-changes-live)
+    - [Your changes are live](#your-changes-are-live)
+    - [Preview form](#preview-form)
+      - [Preview live form](#preview-live-form)
+      - [Preview draft form](#preview-draft-form)
+    - [Form filler view of a live form](#form-filler-view-of-a-live-form)
+
+## User steps
+
```

Comment:
This is very neat but I wonder if it would be better as numbered steps rather than a flow diagram since it's linear?